app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
```

Comment:
Remove the metadata tag, we're going to implement custom fonts and styles once we will reach the first beta release.